src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "src/styles/variables/variables.scss";
+
+@mixin calendar-center {
+  margin: auto;
+}
```

Comment:
`use mix` means not to use `mixin`, you just need to add another class for `calendar` and give margins to it